kroxylicious-operator/src/test/resources/DerivedResourcesTest/colliding-secret-interpolation/out-Deployment-example.yaml

```diff
@@ -10,9 +10,9 @@ kind: "Deployment"
 metadata:
   labels:
     app: "kroxylicious"
+    app.kubernetes.io/part-of: "kafka"
     app.kubernetes.io/managed-by: "kroxylicious-operator"
     app.kubernetes.io/name: "kroxylicious-proxy"
-    app.kubernetes.io/part-of: "kafka"
```

Comment:
I don't think this change is necessary because of the way we do the diffing.

Reply:
ah I see, I think what's happened is that the tests were failing due to other mismatches in the output, but then in the dumped resources IDEA highlighted this as being different. So even if it isn't causing the tests to fail it was a distraction.